modules/ROOT/pages/on-error-scope-concept.adoc

```diff
@@ -63,6 +63,8 @@ to fatal errors, which have error messages that contain the word "fatal":
   type="ANY"
   when='error.cause.message contains "fatal"'/>
 ----
++
+In the example, every error that matches the `when` expression is handled by `on-error-continue`. You can also add a restrictive type such as `type="HTTP:CONNECTIVITY"` so that the error handler only handles the errors that match the specified type and condition. 
```

Comment:
```suggestion
In the example, every error that matches the `when` expression is handled by `on-error-continue`. You can also add a restrictive type such as `type="HTTP:CONNECTIVITY"` so that the error handler handles only the errors that match the specified type and condition. 
```